Dockerfile

```diff
@@ -1,16 +1,10 @@
-# Version: 0.0.3
+# Version: DM-14704
 FROM python:3.6
 LABEL maintainer "maria.t.patterson@gmail.com"
-ENV REFRESHED_AT 2018-01-08
-
-# Install library for confluent-kafka python.
-WORKDIR /home
-RUN git clone https://github.com/edenhill/librdkafka.git && cd librdkafka && git checkout tags/v0.9.4
-WORKDIR /home/librdkafka
-RUN ./configure && make && make install
-ENV LD_LIBRARY_PATH /usr/local/lib
+ENV REFRESHED_AT 2018-06-06
 
 # Pip installs.
+RUN pip install -U pip
```

Comment:
Is this necessary? Maybe it's a standard idiom in Dockerfiles? Might be useful to add a comment explaining.

Reply:
It just complains. I'll take it out.